packages/block-library/src/columns/block.json

```diff
@@ -59,6 +59,14 @@
 			}
 		}
 	},
+	"styles": [
+		{
+			"name": "regular",
+			"label": "Default",
+			"isDefault": true
+		},
+		{ "name": "tablet-stack", "label": "Tablet stack" }
```

Comment:
Just jotting down a brain spasm: I was wondering if we should toggle the style off when `isStackedOnMobile !== true`

It might cause extra confusion though remembering to toggle it back on.